posthog/models/team/team.py

```diff
@@ -202,6 +204,9 @@ def get_effective_membership_level(self, user_id: int) -> Optional["Organization
 
     @property
     def actor_on_events_querying_enabled(self) -> bool:
+        if self.actor_on_events_querying_setting_enabled:
+            return True
+
         # on PostHog Cloud, use the feature flag
         if settings.MULTI_TENANCY:
             return posthoganalytics.feature_enabled(
```

Comment:
Should this feature flag be removed so there's only one source of truth? And we set the `actor_on_events_querying_setting_enabled` flag on all teams that were using the feature flag before?

Reply:
@neilkakkar what do you think about removing all these other conditions?

Reply:
I don't want to remove these, because removing these now means we can't do any rollout at all from our side. And the instance setting depends on the release decision (i.e. do we make person-on-events mandatory on 1.40 or 1.41)

No harm in keeping these for now.

Reply:
Couldn't there be odd behavior where the user tries to use the project setting to disable person-on-enents, but the feature flag is set, so it doesn't work?

Reply:
If we've started rollout on cloud, then yes!

But, I'd say that's intended behaviour, since we don't want people to resist the rollout, whenever it happens? (Although, we'll probs need a better way of explaining what's happening).